hwsc-file-transaction-svc.proto

```diff
@@ -1,37 +1,49 @@
 syntax = "proto3";
 
-package hwscFileTransactionSvc;
+option go_package = "github.com/hwsc-org/hwsc-api-blocks/int/hwsc-file-transaction-svc/file";
+
+package file;
+
+import "lib/authority.proto";
 
 // The file transaction service definition
-service FileTransactionService{
+service FileTransactionService {
     // Gets status for user
-    rpc GetStatus (FileTransactionRequest) returns (FileTransactionResponse){}
+    rpc GetStatus (FileTransactionRequest) returns (FileTransactionResponse) {
+    }
     // Upload files to the storage
-    rpc UploadFile (stream Chunk) returns (FileTransactionResponse){}
+    rpc UploadFile (stream Chunk) returns (FileTransactionResponse) {
+    }
     // Download files from the storage
-    rpc DownloadZippedFiles (FileTransactionRequest) returns (stream Chunk){}
+    rpc DownloadZippedFiles (FileTransactionRequest) returns (stream Chunk) {
+    }
     // Create user folder in the sorage
-    rpc CreateUserFolder(FileTransactionRequest) returns (FileTransactionResponse){}
+    rpc CreateUserFolder (FileTransactionRequest) returns (FileTransactionResponse) {
+    }
 }
 
-message FileTransactionRequest{
-    string message = 1;
-    string name = 2;
-    string uuid = 3;
-    repeated string urls = 4;
+message FileTransactionRequest {
+    lib.Identification identification = 1;
```

Comment:
how to use this lib.Identification identification ?

Reply:
we will have a discussion about this later